bin/postgres-ha/modules/crunchyadm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+username="crunchyadm"
+
+# Create the crunchyadm user if it doesn't already exist
+if [[ "${PGHA_CRUNCHYADM}" == "true" ]] &&
+    ! psql -tAc "SELECT 1 FROM pg_roles WHERE rolname='${username}'" | grep -q 1
+then
+    echo_info "Creating user ${username}"
+    psql -c "CREATE USER ${username} LOGIN;"
```

Comment:
We should be able to use psql variables to do any escaping here.

```sh
psql -v "username=$username" -tAc "SELECT 1 FROM pg_roles WHERE rolname=:'username'"
```

```sh
psql -v "username=$username" -c 'CREATE ROLE :"username" LOGIN'
```

Reply:
Seems like something off with the syntax, any thoughts?

```
bash-4.2$ echo $username
crunchyadm
bash-4.2$ psql -v "username=$username" -tAc "SELECT 1 FROM pg_roles WHERE rolname=:'username'"
ERROR:  syntax error at or near ":"
LINE 1: SELECT 1 FROM pg_roles WHERE rolname=:'username'
```

Reply:
Switched from `-c` to `-f` per discussion to today in order to take advantage of psql variables.

Also went ahead and removed the `grep`, and now just inspecting the output of the `psql` command.